packages/storybook/src/stories/ino-checkbox/ino-checkbox.stories.ts

```diff
@@ -49,43 +58,17 @@ export const Playground: Story<Components.InoCheckbox> = (args) => html`
   </ino-checkbox>
 `;
 
-Playground.args = {
-  checked: false,
-  disabled: false,
-  indeterminate: false,
-  name: '',
-  selection: false,
-  value: '',
-};
+export const Playground = templateForComponent(Template, {});
```

Comment:
Should we think the Template idea further? Lets create a "TemplateGenerator", you pass arguments, a component-tag (`ino-checkbox`) and  a typing. Further, it generates a lit html having the args as attribute names and binding its values.

This would might not be suitable for all elements, but at least for extensive  ones like ino-input, ...